pkg/securitypolicy/securitypolicy.go

```diff
@@ -14,6 +15,12 @@ import (
 	"github.com/pkg/errors"
 )
 
+//go:embed framework.rego
+var FrameworkCode string
```

Comment:
why is this here? shouldn't this be part of rego enforcer?

Reply:
Since the policy simulator can run in either windows or Linux, and since it needs this string to simulate policies, it made sense to move it to securitypolicy.go, which is not boxed by a build tag.